packages/react/src/components/ProgressIndicator/ProgressIndicator.base.tsx

```diff
@@ -23,6 +23,18 @@ export class ProgressIndicatorBase extends React.Component<IProgressIndicatorPro
     width: 180,
   };
 
+  private _id: string;
+  private _labelId: string;
+  private _descriptionId: string;
+
+  constructor(props: IProgressIndicatorProps) {
+    super(props);
+
+    this._id = getId('progress-indicator');
```

Comment:
Just a thought -- does `this._id` actually need to be saved? It looks like only `_labelId` and `_descriptionId` are used later.

Reply:
Yeah, you're right, will fix.